README.md

```diff
@@ -3,10 +3,25 @@
 [![CircleCI](https://circleci.com/gh/solidusio/solidus_braintree.svg?style=shield)](https://circleci.com/gh/solidusio/solidus_braintree)
 [![codecov](https://codecov.io/gh/solidusio/solidus_braintree/branch/master/graph/badge.svg)](https://codecov.io/gh/solidusio/solidus_braintree)
 
-`solidus_braintree` is an extension that adds support for using [Braintree](https://www.braintreepayments.com) as a payment source in your [Solidus](https://solidus.io/) store. It supports Apple Pay, PayPal, and credit card transactions.
+`solidus_braintree` is an extension that adds support for using [Braintree](https://www.braintreepayments.com) as a payment source in your [Solidus](https://solidus.io/) store. It supports Apple Pay, PayPal, Venmo, and credit card transactions. The extension uses Braintree's [JavaScript v3 SDK](https://braintree.github.io/braintree-web/current/).
 
 🚧 This extension is currently only compatible with the `solidus_starter_frontend` 🚧
 
+## Table of Contents
```

Comment:
I'm not a fan of having a manually maintained TOC, as GitHub automatically generates that:

<img src="https://user-images.githubusercontent.com/52650/224228314-97ae8552-e1a5-411c-b1a9-826aed16b0fc.png" width="350" />

However, it's not a feature super well-known, so no strong opinion here :slightly_smiling_face:

Reply:
@waiting-for-dev I wasn't aware about that GitHub feature :p Testing out that TOC, I think it's useful for jumping to a section in the README that you already know. However, it's not as useful as a way to get a high level overview of the README, since it's not large enough to view all of the headings. As such, I think it still makes sense to keep the manually generated table of contents. 

@kennyadsl Do you have any thoughts about the TOC?

Reply:
I also didn't know about this feature, wow! No strong opinions here either, but I'd like to have some sort of consistency across the extensions' README and, the others do not have a TOC. To be fair, it won't be having or not having this TOC to make the README uniform with the other extensions, we will probably need to create a definitive format and update them in bulk to accomplish this, so it's more a thought for later. That said, up to you @gsmendoza, if you like it we can keep it!